[CORL-1749] Live Chat: stop navigation buttons from blocking other stream controls

nick-funk opened the pull request:
## What does this PR do?

Forces flex alignment `div`'s in the live chat navigation buttons from having a height so they don't block click events behind them on the stream.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

- Go to live chat
- View stream when not tailing
- Post a comment with a different account + user
- See the "new comments" or "jump to live" show up
- Line up the button with the "rec", "reply", and "show conversation" buttons of a comment
- Click on the "rec", "reply", or "show conversation" and it will work, no longer blocked by the invisible Flex div that aligns the button center at the bottom of the stream

nick-funk committed:
Prevent Flex align in "jump to" buttons from blocking other stream elements

alinhle reviewed (changes requested):
It looks like it is working from the outside. Looking into the implementation raises a lot of question marks. 

For the main view I see 3 flex container elements, which seems excessive and redundant, and makes the `noHeight` unnecessary as well. For the conversation view there is still 2 flex container elements. The use of `overflow: visible` and `height: 0px` is also questionable in general. 

![image](https://user-images.githubusercontent.com/14221600/116418015-0d535000-a83c-11eb-881c-0501e3c444bf.png)

I recommend combining `position: relative`, `position: absolute`, `pointer-events: none`, `pointer-events: initial` for a clean implementation, removing the additional flex container inside `LiveChat` and refactor all jump buttons to use `JumpToButton` inside of the conversation view.

nick-funk committed:
Position "jump to" buttons without multiple flex alignments

- Use absolute position + transform to center the element
- Remove duplicate Flex boxes
- Refactor remaining "jump to" button that was not using the
new JumpToButton in Conversation view

nick-funk commented:
> It looks like it is working from the outside. Looking into the implementation raises a lot of question marks.
> 
> For the main view I see 3 flex container elements, which seems excessive and redundant, and makes the `noHeight` unnecessary as well. For the conversation view there is still 2 flex container elements. The use of `overflow: visible` and `height: 0px` is also questionable in general.
> 
> ![image](https://user-images.githubusercontent.com/14221600/116418015-0d535000-a83c-11eb-881c-0501e3c444bf.png)
> 
> I recommend combining `position: relative`, `position: absolute`, `pointer-events: none`, `pointer-events: initial` for a clean implementation, removing the additional flex container inside `LiveChat` and refactor all jump buttons to use `JumpToButton` inside of the conversation view.

I didn't need to do `pointer-events` to make this work, but I was able to remove the Flex's and make this work properly using `position: relative` and `position: absolute` with a `transform` to help center the element dynamically. Should be a lot cleaner now.

alinhle committed:
chore: simplify jümp to büttons
